### ISSUE TITLE
v3.2.1 Fails on indexing while migrating

### ISSUE THREAD

**Issue by tim-hanssen** [neo4j/neo4j#9498]
We are stuck in the same error as described in https://github.com/neo4j/neo4j/issues/9331 while migrating our instance to 3.2.1. As far as we know, there are no parameters indexed that are that long. Still we got this error.

It looks like the Tag(hash) index fails, but the longest string in that param is 32 chars. Any ideas on how we can find the node that is breaking our import?

2017-06-09 10:31:55.716+0000 ERROR [o.n.k.i.a.i.IndexPopulationJob] Failed to populate index: [:Tag(Hash) [provider: {key=lucene, version=1.0}]] Property value bytes length: 35389 is longer then 32766, which is maximum supported length of indexed property value.
java.lang.IllegalArgumentException: Property value bytes length: 35389 is longer then 32766, which is maximum supported length of indexed property value.
at org.neo4j.kernel.impl.api.IndexValueLengthValidator.validate(IndexValueLengthValidator.java:44)
at org.neo4j.kernel.impl.api.IndexSimpleValueValidator.validate(IndexSimpleValueValidator.java:42)
at org.neo4j.kernel.impl.api.IndexValueValidator.validate(IndexValueValidator.java:36)
at org.neo4j.kernel.impl.transaction.state.storeview.StoreViewNodeStoreScan.process(StoreViewNodeStoreScan.java:108)
at org.neo4j.kernel.impl.transaction.state.storeview.NodeStoreScan.run(NodeStoreScan.java:74)
at org.neo4j.kernel.impl.api.index.BatchingMultipleIndexPopulator$BatchingStoreScan.run(BatchingMultipleIndexPopulator.java:378)
at org.neo4j.kernel.impl.api.index.IndexPopulationJob.indexAllNodes(IndexPopulationJob.java:137)
at org.neo4j.kernel.impl.api.index.IndexPopulationJob.run(IndexPopulationJob.java:109)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
at org.neo4j.helpers.NamedThreadFactory$2.run(NamedThreadFactory.java:109)

**Comment by jexp** [comment]
did you try to run

```
MATCH (t:Tag) 
RETURN size(t.Hash), count(*)

```
or 

```
MATCH (t:Tag) 
WHERE size(t.Hash) >  30000
RETURN t
```

@MishaDemianenko we should probably also make the error message more informative, also reporting the node-id ?

**Comment by tim-hanssen** [comment]
Hey @jexp,

Thxs for your comment.
When we run that query:

size(t.Hash)	count(*)
32	1014

The second query returns ofc no results.
(no rows)

**Comment by tim-hanssen** [comment]
We fixed the issue by updating first to 3.0.7, 3.1.4  and finally to 3.2.1.
